styles/main.css

```diff
@@ -88,13 +91,312 @@ ul {
   list-style-type: none;
 }
 
-section ul {
+article ul {
+  width: 9.75rem;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 1.5rem;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: auto;
+  width: 375px;
+  border-radius: 16px;
+  padding: 16px;
+}
+
+.content {
+  margin-top: 248px;
+  margin-right: 16px;
+  margin-left: 16px;
+  margin-bottom: 12px;
+  height: 80px;
+}
+
+.content h4 {
+  width: 295px;
+  height: 36px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin: 12px 0;
+}
+
+.grid-item {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  background-color: #fff;
+  color: #344563;
+  background-repeat: no-repeat;
+  background-position: 16px 16px;
+  border-radius: 16px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 12px 16px;
+  width: 327px;
+  height: 552px;
+}
+
+.grid-item ul {
+  width: 203px;
+  height: 24px;
+  padding-left: 0;
+}
+
+.content ul {
+  list-style-type: disc;
+  justify-content: space-between;
+  width: 236px;
+  padding-left: 0;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.content li {
+  color: #7a869a;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.ocontent li {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  margin-left: 16px;
+  background-color: #ebebff;
+  color: #6070ff;
+  border-radius: 8px;
+  padding: 6px 12px;
+  align-items: center;
+}
+
+footer {
+  background-color: #6070ff;
+  color: white;
+  background-image: url(/imgs/contact-form\ shapes-mobile.png);
```

Comment:
- [ ] Please make sure you have no spaces in the file name of this background image. You may also check this [article](https://www.w3schools.com/css/css3_backgrounds.asp) for how to add multiple background images because there are two background images for the footer section in the Figma design.

Reply:
@gtekle , This is already implemented. Thank you for the kind note

Reply:
Great job! 👍🏾